config/SingleCaps.robot

```diff
@@ -0,0 +1,2 @@
+*** Variables ***
+&{SINGLE_CAPS}      name=single test    build=browserstack-robot-framework    browser=Edge    browser_version=latest    os=Windows    os_version=10
```

Comment:
For all sample repos we are keeping the build name as `browserstack-build-1`, let's update this for local and parallel as well.

Reply:
done under commit da231388133e9a65e54023ae9d56dbefaf500daf